com.unity.render-pipelines.high-definition/Runtime/ShaderLibrary/ShaderVariables.hlsl

```diff
@@ -156,6 +156,12 @@ TEXTURE2D(_PrevExposureTexture);
 
 #include "Packages/com.unity.render-pipelines.high-definition/Runtime/ShaderLibrary/ShaderVariablesXR.cs.hlsl"
 
+#ifdef SAMPLE_TEXTURE2D
+#undef SAMPLE_TEXTURE2D
+#define SAMPLE_TEXTURE2D(textureName, samplerName, coord2) \
```

Comment:
Is this what we want?
Unless I'm mistaken, doing that will cause all samples 2D to apply the bias, including internal render loop stuff. 
Thinking of stuff like color pyramid which has mips etc. There might not be many such textures but it could break things I guess.
The alternative is not super nice though. It means that we'd need to update all our hand written + SG shaders to use SAMPLE_TEXTURE2D_BIAS with the global bias instead?

Reply:
Yes, so what's happening is that in the CPU (C# side) i reset the bias to 0 always, after material rendering (I have to double check that).
  
So color pyramid / other passes will remain with bias == 0.
As you said, the alternative is to be explicit into every shader. We can discuss this perhaps with Seb.

Reply:
From what I see we update the global CB at the beginning of the camera render once with the current global mip bias and then only resets it at the end before posts. Am I missing something?

Reply:
@JulienIgnace-Unity I have updated the review with the proper changes. Could you verify? I have only done this during Prepass (that is gbuffer laydown).
I will have to rework this on the actual DLSS change and verify mip bias against hte missing passes. This time around I rather just do it on a couple of passes before going wide.
Let me know.

Reply:
it would be worth it to add that type of info to the documentation btw. what is affected by the feature?

Reply:
I can add some documentation. Every texture sampler is affected. The only big difference is the fact that you can now pass a bias. If the bias is 0 though, you get the exact same behaviour.